### ISSUE TITLE
No more battle music after encountering random shadow monsters

### ISSUE THREAD

**Issue by Pronyo-Chan** [MarvinXLII/OT2R#5]
After I had my first random encounter with the weird shadow beings where the usual battle theme does not play and instead keeps playing the eerie theme, the battle themes are just gone now. No matter the area I have a random encounter in, the BGM just goes silent during the fight, until the battle ends and the field theme starts playing back.

It seems to be save-file related, as even restarting the game does not fix it and save files from previous runs don't have the issue..

**Comment by Pronyo-Chan** [comment]
After opening my world map, I realized what happened. Since I obtained the "Sacred Flame Extinguished: Wardenbeasts' Tomb", that tomb is now enveloped in purple mist and I must light the flame. The game now considers I'm at the endgame, where the battle music indeed does not play at that point...

I will close the issue as it is not a bug per say, but I think the battle music should ideally not be affected by obtaining these items. It's very confusing and I now have to defeat a boss in a lvl51 area to bring the music back I assume, which is gonna take me a while as I got the item pretty early.
